app/styles/style.css

```diff
@@ -131,4 +137,289 @@ body {
     text-decoration: none;
     display: flex;
   }
+
+  .jobs {
+    padding: 24px;
+  }
+
+  .card {
+    display: flex;
+    align-items: flex-start;
+    flex-wrap: wrap;
+    flex-direction: column;
+    height: auto;
+    width: 100%;
+    background-color: #fff;
+    border-radius: 16px;
+    border: 1px #dfe1e6 solid;
+    margin-top: 114px;
+  }
+
+  .card-container {
+    padding: 16px;
+  }
+
+  .card .card-img {
+    width: 100%;
+  }
+
+  .card-title-text {
+    margin: 0;
+    margin-top: 16px;
+  }
+
+  .job-img {
+    border-radius: 8px;
+    width: 100%;
+  }
+
+  .card-title {
+    font-family: PoppinsBold;
+    font-size: 24px;
+    line-height: 44px;
+    font-weight: 700;
+  }
+
+  .card-subtitle {
+    display: grid;
+    grid-template-columns: auto auto auto auto auto;
+    justify-content: space-between;
+    align-items: center;
+    width: 236px;
+    font-family: Poppins;
+  }
+
+  .el1 {
+    color: #344563;
+    font-size: 13px;
+    line-height: 16px;
+    font-weight: 600;
+  }
+
+  .el2 {
+    color: #7a869a;
+    font-size: 13px;
+    line-height: 16px;
+    font-weight: 600;
+  }
+
+  .dividor {
+    background-color: #c1c7d0;
+    width: 8px;
+    height: 8px;
+    border-radius: 50%;
+  }
+
+  .card-body {
+    width: 100%;
+    margin: 0;
+  }
+
+  .card-body p {
+    margin-top: 4px;
+  }
+
+  .card-description {
+    font-family: Poppins;
+    color: #344563;
+    font-size: 15px;
+    font-weight: 400;
+    line-height: 24px;
+  }
+
+  .stack-section {
+    display: flex;
+  }
+
+  .stack-container {
+    padding-right: 8px;
+  }
+
+  .tech-stack {
+    color: #6070ff;
+    background-color: #ebebff;
+    font-family: Poppins;
+    height: 24px;
+    padding: 6px 12px 6px 12px;
+    font-size: 12px;
+    align-content: center;
+    border-radius: 8px;
+  }
+
+  .card-footer {
+    margin-top: 48px;
+    padding-bottom: 16px;
+  }
+
+  .show-button {
+    padding: 12px;
+    text-decoration: none;
+    border: 1px #6070ff solid;
+    border-radius: 8px;
+    color: #6070ff;
+    font-family: Poppins;
+  }
+
+  .show-button:hover {
+    background-color: #6070ff;
+    box-shadow: 0 8px 16px 0 rgba(64, 83, 252, 0.25);
+    color: #fff;
+  }
+
+  .show-button:active {
+    background-color: #2230d2;
+    color: #fff;
+  }
+
+  .about-section {
+    display: flex;
+    flex-direction: column;
+    margin-top: 114px;
+    background-color: #fff;
+    border-radius: 0 15% 0 0;
+    padding: 24px;
+  }
```

Comment:
- `[OPTIONAL]` Kindly try to make your design as much as closer to the `Figma` design because it is important to carefully look at the small details which indeed makes you not only a better developer but also make you a `detail-oriented` developer. Kindly use  `border-radius: 0 75px 0 0;` instead of ` border-radius: 0 15% 0 0;` and also use the correct `font-weight`.

<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-04-29 13-20-29](https://user-images.githubusercontent.com/66476329/235292703-1908a2a0-0523-4a18-a3bb-dbbb4791cab1.png)

</td>
<td>

![Screenshot from 2023-04-29 13-19-40](https://user-images.githubusercontent.com/66476329/235292707-b2edb3cd-7341-4052-8c83-91a4293476fe.png)

</td>
</tr>
</table>

Reply:
I gave the title a with of 24px but it was supposed to be 40px. it on the .about-title selector.